.travis.yml

```diff
@@ -45,5 +45,6 @@ jobs:
       script:
         - ./vendor/bin/phpunit --exclude-group performance --coverage-clover clover.xml
       after_script:
-        - wget https://scrutinizer-ci.com/ocular.phar
-        - php ocular.phar code-coverage:upload --format=php-clover clover.xml
+        - wget https://github.com/php-coveralls/php-coveralls/releases/download/v2.2.0/php-coveralls.phar --output-document="${HOME}/bin/coveralls"
+        - chmod +x ${HOME}/bin/coveralls
+        - travis_retry coveralls -v --json_path /tmp/coveralls-upload.json --coverage_clover=clover.xml
```

Comment:
@morozov I thought it would be more clever to put everything in the same step this time…
- easier to understand
- coveralls is not required to run the tests so shouldn't really be under `install`
- also, I noticed that I forgot to add `.coveralls` in `.gitattributes` on the DBAL, and thought that it might just be best not to have have a config file at all

I will open a PR on the DBAL if the above is ok with you.